internal/display/logs.go

```diff
@@ -149,12 +149,15 @@ func (r *Renderer) LogList(logs []*management.Log, silent, hasFilter bool) {
 
 	if len(logs) == 0 {
```

Comment:
Suggestion:

```go
if len(logs) == 0 {
	if !hasFilter {
		r.EmptyState(resource, "To generate logs, run a test command like 'auth0 test login' or 'auth0 test token'")
		return
	}
	
	r.EmptyState("logs matching filter criteria", "")
	return
}
```

Like this it will print "No logs matching filter criteria available." and we have less complex code.

Reply:
IMO that reads very poorly. I'm glad we were able to reduce a couple one-off instances, but I think this one still makes sense.

Reply:
Both sentences are grammatically correct and convey the same meaning 🤷🏻‍♂️ Not a blocker tho.